internal/actioninfo/actioninfo.go

```diff
@@ -1,9 +1,31 @@
 package actioninfo
 
+import (
+	"fmt"
+
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DataParser interface {
 	// TODO: добавить методы
+	Parse(datastring string) error
+	ActionInfo() (string, error)
 }
 
 func Info(dataset []string, dp DataParser) {
 	// TODO: реализовать функцию
+	for _, v := range dataset {
+		err := dp.Parse(v)
+		if err != nil {
+			fmt.Println(spentenergy.ErrData)
```

Comment:
1. Теряешь сообщение ошибки
2. Эта ошибка из другого пакета, лучше их не мешать
```suggestion
			fmt.Println("error in Parsing:", err)
```